app/services/weet.js

```diff
@@ -1,12 +1,25 @@
 const axios = require('axios');
 const { url } = require('../../config').common.weetService;
-const { defaultError } = require('../errors');
+const { defaultError, databaseError } = require('../errors');
+const { Weet } = require('../models');
+const logger = require('../logger');
+const { WEET_API_ERROR } = require('../constants/errors');
+exports.getWeetContent = async () => {
+  try {
+    const { data: { joke } } = await axios.get(`${url}?format=json`);
+    return joke;
+  } catch (error) {
+    logger.error(error);
+    throw defaultError(WEET_API_ERROR);
```

Comment:
Although `500` is ok, it is too generic. So, create a new custom error which responds with `503`